packages/richtext-lexical/src/features/blocks/server/index.ts

```diff
@@ -26,21 +33,46 @@ export const BlocksFeature = createServerFeature<BlocksFeatureProps, BlocksFeatu
         {
           name: 'lexical_blocks',
           type: 'blocks',
-          blocks: props.blocks ?? [],
+          blockReferences: _props.blocks ?? [],
+          blocks: [],
         },
         {
           name: 'lexical_inline_blocks',
           type: 'blocks',
-          blocks: props.inlineBlocks ?? [],
+          blockReferences: _props.inlineBlocks ?? [],
+          blocks: [],
         },
       ],
       parentIsLocalized,
       requireFieldLevelRichTextEditor: isRoot,
       validRelationships,
     })
 
-    props.blocks = (sanitized[0] as BlocksField).blocks
-    props.inlineBlocks = (sanitized[1] as BlocksField).blocks
+    const pureBlocks: Block[] = []
```

Comment:
`pureBlocks`????? ?!!!!!! 

maybe `blockConfigs`? implying that it's a config and not a string ref